core/src/main/java/org/lflang/LinguaFranca.xtext

```diff
@@ -237,6 +237,7 @@ Instantiation:
     ')' (('at' host=Host ';') | ';'?);
 
 Connection:
+    (attributes+=Attribute)*
```

Comment:
What's the role of this change?

Reply:
Nice catch! That is a mistake and has nothing to do with this PR, thank you!